frontend/src/components/layout/Header/Header.styled.ts

```diff
@@ -45,15 +66,41 @@ export const BlueCasperLogo = styled.img`
   }
 `;
 
-export const HeroContainer = styled.div`
+export const LogoText = styled.span`
+  font-family: lausanne;
+  font-size: ${pxToRem(26)};
+  font-weight: ${fontWeight.medium};
+  letter-spacing: ${pxToRem(-1)};
+  padding-top: ${pxToRem(6)};
+  position: absolute;
+  left: ${pxToRem(119)};
+  white-space: nowrap;
+  background-image: linear-gradient(
+    90deg,
+    #1c1e90,
+    #693590,
+    #d81d54,
+    #d81e54,
+    #fd6b52
+  );
+  background-size: 100%;
+  background-clip: text;
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  -moz-text-fill-color: transparent;
+`;
+
+export const HeroContainer = styled.div<{ isFirstVisit: boolean }>`
   display: flex;
   justify-content: center;
   padding-top: 2rem;
 
   @media (min-width: ${breakpoints.lg}) {
-    padding: 8rem 6rem 0rem 0rem;
-    width: 100%;
-    justify-content: center;
+    justify-content: start;
+    width: 64.5%;
+    max-width: ${pxToRem(792)};
```

Comment:
Do we need a max width on this?

Reply:
The max-width limits the growth of the container once as the page is resized.  Because their isn't a general wrapper the element, 65% becomes too large once the vw is beyond 1200 px.